specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/common-types.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
why not use https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json ?

Reply:
There are some differences between types.json and the ones we are using in cost-management. I tried using ApiVersionParameter from types.json and breaking changes got flagged - https://dev.azure.com/azure-sdk/590cfd2a-581c-4dcb-a12e-6568ce786175/_apis/build/builds/1505816/logs/242.

We have more detailed description in ErrorResponse property and our ProxyResource doesn't have SystemData. Hence, we cannot use types.json for the ones we have defined in common-types.json.